styleguide/derek/view-templates/leadership/_leadership.scss

```diff
@@ -0,0 +1,53 @@
+.leadership-individual {
+  @include make-sm-column(3);
+  @include make-xs-column(6);
+  margin: 20px auto;
+}
+
+.leadership-image {
+  @include center-block;
+}
+
+.leadership-imageWrapper {
+  display: none;
+}
+
+.leadership-linkWrapper {
+  margin-top: 0;
+}
+
+.leadership-attrWrapper {
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.leadership-name {
+  margin-bottom: 0;
+}
+
+.leadership-title {
+  margin: 0;
+}
+
+.leadership-link {
+  @include standard-link;
+}
+
+@media only screen and (min-width: $screen-xs-max) {
+  .leadership-individual {
+    &.center-block {
+      float: none;
```

Comment:
Well that brings up a good point.  There should be no layout mandated within the pattern itself.  So `leadership-individual` shouldn't set a width at all (meaning it wouldn't @include anything). 

Then inside each row in the `leadership.ejs`, we need to setup two new layouts that would also have to get created as part of the www ticket. We'd need the following layouts:
- Mobile 2-up that expands to a 4-up on desktop/table
- Mobile centered 1-up that is 50% wide, that goes to 25% on table/desktop. 

Layouts are fine to use core bootstrap classes as we already have that precedent set.